vsintegration/src/vs/FsPkgs/FSharp.VS.FSI/fsiSessionToolWindow.fs

```diff
@@ -74,12 +74,17 @@ module internal Locals =
 open Util
 open Locals
 
-// consumed by C#, so enum type used instead of union
+// consumed by C#, so enum types used instead of unions
 type internal FsiDebuggerState =
     | NotRunning = 0
     | AttachedToFSI = 1
     | AttachedNotToFSI = 2
 
+type internal FsiEditorSendAction =
```

Comment:
Is there any risk in making this type a DU type? In the end, you just create a simple union case, don't you?